src/azure-cli/azure/cli/command_modules/storage/azcopy/util.py

```diff
@@ -56,7 +56,8 @@ def install_azcopy(self, install_location):
         install_dir = os.path.dirname(install_location)
         if not os.path.exists(install_dir):
             os.makedirs(install_dir)
-        base_url = 'https://azcopyvnext.azureedge.net/release20211027/azcopy_{}_{}_{}.{}'
+        base_url = 'https://azcopyvnext-awgzd8g7aagqhzhe.b02.azurefd.net/releases/release-10.27.1-20241113/' \
```

Comment:
Probably we can get the download pkg from short link like `https://aka.ms/downloadazcopy-v10-windows`/`https://aka.ms/downloadazcopy-v10-linux`/`https://aka.ms/downloadazcopy-v10-...`. The only thing we need to confirm is that if we can work well with latest azcopy version. If so, there's no need to use a fixed azcopy version

Reply:
The latest azcopy version 10.27.1 has memory leak issues that have been reported here
https://github.com/Azure/azure-storage-azcopy/issues/2855
Reverting back to version 10.13.0 and users can manually install any newer version if needed.